http/http-advanced/src/test/java/io/quarkus/ts/openshift/http/AbstractHttpTest.java

```diff
@@ -97,14 +102,8 @@ public void testGrpc() {
     @Test
     @DisplayName("Http/2 Server test")
     public void http2Server() throws InterruptedException, URISyntaxException {
```

Comment:
Rename test case to `vertxHttpClientRedirectionUsingHttp2` and update the `@DisplayName` accordingly for this method and the other too.

Reply:
No, here we are not testing anything related to VertxHttpClient. The goal of this test is to test that server-side support http/2.